Packs/PAN-OS/Playbooks/PAN-OS_-_Job_-_Add_Malicious_Domains_To_Sinkhole.yml

```diff
@@ -0,0 +1,1524 @@
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "50":
+    id: "50"
+    taskid: 666a7f2c-1788-4595-8b56-04f9da182103
+    type: regular
+    task:
+      id: 666a7f2c-1788-4595-8b56-04f9da182103
+      version: -1
+      name: Save their corresponding domains in a new key
+      description: Saves the domains that have relationships to threat actors in a new key.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      key:
+        simple: DomainsRelatedToThreatActors
+      value:
+        complex:
+          root: RelationshipsWithCampaigns
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: RelationshipsWithCampaigns.EntityBType
+                iscontext: true
+              right:
+                value:
+                  simple: Threat Actor
+              ignorecase: true
+          accessor: EntityA
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1170,
+          "y": 2150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "15_14_#default#": 0.2,
+      "15_24_yes": 0.35,
+      "17_14_#default#": 0.33,
+      "17_18_yes": 0.54,
+      "18_14_#default#": 0.44,
+      "24_14_#default#": 0.24,
+      "24_20_yes": 0.47,
+      "26_29_#default#": 0.16,
+      "30_31_#default#": 0.24,
+      "30_4_yes": 0.4,
+      "35_36_yes": 0.5,
+      "35_8_#default#": 0.16,
+      "37_41_yes": 0.47,
+      "37_8_#default#": 0.2,
+      "46_48_yes": 0.4,
+      "46_8_#default#": 0.1,
+      "49_50_yes": 0.35,
+      "49_8_#default#": 0.1,
+      "4_14_#default#": 0.1
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3400,
+        "width": 3140,
+        "x": -990,
+        "y": 35
+      }
+    }
+  }
+inputs:
+- key: ""
+  value: {}
+  required: false
+  description: All domain indicators. In the playbook, the domains will be filtered by those used for malicious communication, and tagged to be sinkholed.
+  playbookInputQuery:
+    query: type:Domain
+    queryEntity: indicators
+    results:
+    daterange:
+      fromdate: 0001-01-01T00:00:00Z
+      todate: 0001-01-01T00:00:00Z
+      period:
+        by: ""
+        byto: ""
+        byfrom: ""
+        tovalue:
+        fromvalue:
+        field: ""
+      fromdatelicenseval: 0001-01-01T00:00:00Z
+    runFromLastJobTime: true
+- key: SinkholeTagForEDL
+  value:
+    simple: to_sinkhole
+  required: true
+  description: The tag that should be applied to the domain so that it will be exported to the EDL using the Generic Export Indicators Service integration in XSOAR.
```

Comment:
```suggestion
  description: The tag that should be applied to the domain so that it will be exported to the EDL using the Generic Export Indicators Service integration in Cortex XSOAR.
```